pages/reset-password.js

```diff
@@ -5,11 +5,16 @@ import BackButton from '@/components/BackButton';
 import PasswordResetForm from '@/components/forms/PasswordReset';
 
 import logo from '@/public/logo.svg';
+import bellIcon from '@/public/bell-icon.svg';
+import Alert from '@/components/Alert';
 
 export const passwordResetPageTestIds = {
   headlineText: 'headline-text',
 };
 
+const ALERT_TITLE = 'Check your email';
+const ALERT_DESCRIPTION = "We've emailed you instructions to reset your password";
```

Comment:
[opinionated comment — feel free to reject :D]
It is not needed at the moment, but I cannot stop thinking about 

> "what is the provided email has an invalid format? We would then have a second alert to indicate that the email is invalid... 🤔  
> Then these const name would be in conflict". 

A way to anticipate this issue is to have a more `event specific` name (and less `component specific`) for these constants. 
For example:

```suggestion
const SUCCESS_TITLE = 'Check your email';
const SUCESS_DESCRIPTION = "We've emailed you instructions to reset your password";
```

Note that this name also does **not** care what component will be used to display the message. This is somehow important if later on the "alert" component is renamed (or replace) — which reduces the risk to forget to update this name and endup with something like `<UserMessage title={ALERT_TITLE} ... />`

Reply:
I got your point, event specific names chosen in 53d7ef1afdbb971283c5f0ce7fac8c42eabe7e1c